climakitae/core/data_export.py

```diff
@@ -561,14 +568,13 @@ def _export_to_csv(data, save_name):
     # Check file size and avail workspace disk space
     # raise error for not enough space
     # and warning for large file
-    file_size_threshold = 0.85  # in GB
+    est_file_size = _estimate_file_size(data, "CSV")
     disk_space = shutil.disk_usage(os.path.expanduser("~"))[2] / bytes_per_gigabyte
-    data_size = data.nbytes / bytes_per_gigabyte
 
-    if disk_space <= data_size:
+    if disk_space <= est_file_size:
         raise Exception(
             "Not enough disk space to export data! You need at least "
-            + str(round(data_size, 2))
+            + str(round(est_file_size, 2))
             + (
                 " GB free in the hub directory, which has 10 GB total space."
                 " Try smaller subsets of space, time, scenario, and/or"
```

Comment:
I would suggest replacing "clean" with "delete" since I took me a second to understand if this was cleaning (deleting) or cleaning (data cleaning).

Reply:
Agreed. Fixed in 636de95